ext/ffi/jit_trampoline.rs

```diff
@@ -83,20 +94,35 @@ pub(crate) fn codegen(sym: &crate::Symbol) -> String {
     c += native_arg_to_c(ty);
     let _ = write!(c, " p{i}");
   }
+  if needs_unwrap {
+    let _ = write!(c, ", struct FastApiTypedArray* const p_ret");
```

Comment:
The inital comma here.. what if there are no parameters, but a return type that needs an unwrap?

Reply:
well there is always a `recv`

Reply:
Nevermind, there's an unconditional receiver to start.